modules/statics/src/networks.ts

```diff
@@ -677,13 +677,13 @@ class HashTestnet extends Testnet implements AccountNetwork {
 class Bld extends Mainnet implements AccountNetwork {
   name = 'Agoric';
   family = CoinFamily.BLD;
-  explorerUrl = 'https://bigdipper.live/agoric/transactions/';
+  explorerUrl = 'https://agoric.explorers.guru/transaction/';
 }
 
 class BldTestnet extends Testnet implements AccountNetwork {
   name = 'Testnet Agoric';
   family = CoinFamily.BLD;
-  explorerUrl = 'https://emerynet.explorer.agoric.net/agoric/tx/';
+  explorerUrl = 'https://devnet.agoric.explorers.guru/transaction/';
```

Comment:
Why do you want to point on devnet ? we are using emerynet for testing.
```suggestion
  explorerUrl = 'https://devnet.agoric.explorers.guru/transaction/';
```

Reply:
Apologies for the confusion, I accidentally replaced the Emerynet chain explorer URL with the Devnet chain explorer URL in the configs file. I'll revert this change promptly.